src/main/java/com/example/todolist/API/Restful/Dto/Request/CreateTaskRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.example.todolist.API.Restful.Dto.Request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+public record CreateTaskRequest (
+        @NotNull
+        @NotBlank
```

Comment:
Create a rule called TitleRule and use it instead, this is not the same case as Password